shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/BlockChain.hs

```diff
@@ -637,18 +636,20 @@ checkVRFValue certNat σ f =
     -- bet set above one half otherwise.
       True
     else
-      if leaderVal == 1
+      if certNat >= certNatMax
```

Comment:
Can't we eliminate this special case? If we have a 256bit VRF output, then it is in the range 0..2^256-1, so if we take `certNatMax` as 2^256 then `certNatMax - certNat` is at minimum 1, and hence no division by zero and no special case.

Reply:
yes indeed, that's nice!

Reply:
I think it's still useful to have a check that the value is within the range we say it is. But yeah, I guess here isn't necessarily the best place for it.

Reply:
oh I see, it's just a natural. and indeed below in a unit test we make `vrfOne`.